hc/api/tests/test_send_reports.py

```diff
@@ -0,0 +1,57 @@
+from datetime import timedelta
```

Comment:
These tests don't seem to be testing the functionality you added (weekly and daily reports)